aws-xray/src/test/groovy/io/micronaut/tracing/aws/XRayConfigurationSpec.groovy

```diff
@@ -0,0 +1,100 @@
+package io.micronaut.tracing.aws
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.env.Environment
+import io.micronaut.tracing.aws.client.SdkClientBuilderListener
+import io.micronaut.tracing.aws.cloudwatch.MetricsSegmentListenerFactory
+import io.micronaut.tracing.aws.filter.XRayHttpServerFilter
+import spock.lang.Specification
+
+class XRayConfigurationSpec extends Specification {
+
+    def "all integrations are enabled"() {
+        given:
+        ApplicationContext applicationContext = ApplicationContext.run([
+                "micronaut.application.name": "test-application",
+        ], Environment.AMAZON_EC2)
+
+        when:
+        XRayConfiguration.XRayHttpFilterConfiguration httpFilterConfiguration = applicationContext.getBean(XRayConfiguration.XRayHttpFilterConfiguration)
+        XRayConfiguration.SdkClientsConfiguration sdkClientsConfiguration = applicationContext.getBean(XRayConfiguration.SdkClientsConfiguration)
+        XRayConfiguration.XRayCloudWatchMetricsConfiguration cloudWatchMetricsConfiguration = applicationContext.getBean(XRayConfiguration.XRayCloudWatchMetricsConfiguration)
+
+        then:
+        httpFilterConfiguration.isEnabled()
+        sdkClientsConfiguration.isEnabled()
+        cloudWatchMetricsConfiguration.isEnabled()
+    }
+
+    def "it configures sampling rule"(){
+        given:
+        ApplicationContext applicationContext = ApplicationContext.run([
+                "micronaut.application.name" : "test-application",
+                "aws.xray.samplingRule": "rule"
```

Comment:
Micronaut configuration is kebap case. 

This should be `aws.xray.sampling-rule`

Reply:
I've changed this.